build.gradle

```diff
@@ -180,6 +180,7 @@ okbuck {
         espressoForLibraries = true
         robolectric = true
         robolectricApis = ["21", "27"]
+        enableIntegrationTests = true
```

Comment:
probably name it `integrationTestForJava`

Reply:
The current work is for all jvm targets, so it should work out of the box. I've tested with kotlin, for example, and was able to setup integration tests for it as well